src/loaders/ncbi_downloader/ncbi_downloader.py

```diff
@@ -0,0 +1,73 @@
+"""
+Download genome files from NCBI FTP server
+"""
+
+import os
+import time
+from datetime import datetime
+from typing import List
+import ftputil
+import pytz
+
+
+def _download_genome_file(download_dir: str, gene_id: str, target_file_ext: List[str]) -> None:
+    ncbi_domain, user_name, password = 'ftp.ncbi.nlm.nih.gov', 'anonymous', 'anonymous@domain.com'
+
+    success, attempts, max_attempts = False, 0, 3
+    while attempts < max_attempts and not success:
+        try:
+            time.sleep(5 * attempts)
+            with ftputil.FTPHost(ncbi_domain, user_name, password) as host:
+
+                gene_dir = '/genomes/all/{}/{}/{}/{}/'.format(
+                    gene_id[0:3], gene_id[4:7], gene_id[7:10], gene_id[10:13])
+
+                host.chdir(gene_dir)
+                dir_list = host.listdir(host.curdir)
+                gene_dir_name = [i for i in dir_list if i.startswith(gene_id)][0]
```

Comment:
What other directories are in this directory other than the one we want?

Reply:
other versions of the genome